src/components/navigation/Hamburger/styles.css

```diff
@@ -0,0 +1,144 @@
+#menuToggle
+{
+  display: inline ;
+  position: absolute;
+  top: 25px;
+  left: 25px;
+  
+  z-index: 1;
+  
+  -webkit-user-select: none;
+  user-select: none;
+}
+
+#menuToggle a
```

Comment:
You never know how much the app is going to grow so try to give a class name to every element in your component, not only for styling but for making it more readable and specific.

Reply:
Good call, will do